tests/test_postgres_discovery.py

```diff
@@ -0,0 +1,473 @@
+            'invalid_tsquery',
+            'invalid_tsvector',
+            'invalid_txid_snapshot',
+            'invalid_xml',
+        }
+    @staticmethod
+    def expected_schema_types():
+        return {
+            'id': 'integer',  # 'serial primary key',
+            'our_varchar': 'character varying',  # 'varchar'
+            'our_varchar_10': 'character varying',  # 'varchar(10)',
+            'our_text': 'text',
+            'our_text_2': 'text',
+            'our_integer': 'integer',
+            'our_smallint': 'smallint',
+            'our_bigint': 'bigint',
+            'our_decimal': 'numeric',
+            'OUR TS': 'timestamp without time zone',
+            'OUR TS TZ': 'timestamp with time zone',
+            'OUR TIME': 'time without time zone',
+            'OUR TIME TZ': 'time with time zone',
+            'OUR DATE': 'date',
+            'our_double': 'double precision',
+            'our_real': 'real',
+            'our_boolean': 'boolean',
+            'our_bit': 'bit',
+            'our_json': 'json',
+            'our_jsonb': 'jsonb',
+            'our_uuid': 'uuid',
+            'our_store': 'hstore',
+            'our_citext': 'citext',
+            'our_cidr': 'cidr',
+            'our_inet': 'inet',
+            'our_mac': 'macaddr',
+            'our_alignment_enum': 'alignment',
+            'our_money': 'money',
+            'invalid_bigserial': 'bigint',
+            'invalid_bit_varying': 'bit varying',
+            'invalid_box': 'box',
+            'invalid_bytea': 'bytea',
+            'invalid_circle': 'circle',
+            'invalid_interval': 'interval',
+            'invalid_line': 'line',
+            'invalid_lseg': 'lseg',
+            'invalid_path': 'path',
+            'invalid_pg_lsn': 'pg_lsn',
+            'invalid_point': 'point',
+            'invalid_polygon': 'polygon',
+            'invalid_serial': 'integer',
+            'invalid_smallserial': 'smallint',
+            'invalid_tsquery': 'tsquery',
+            'invalid_tsvector': 'tsvector',
+            'invalid_txid_snapshot': 'txid_snapshot',
+            'invalid_xml': 'xml',
+        }
+
+    @staticmethod
+    def tap_name():
+        return "tap-postgres"
+
+    @staticmethod
+    def name():
+        return "tap_tester_postgres_discovery"
+
+    @staticmethod
+    def get_type():
+        return "platform.postgres"
+
+    @staticmethod
+    def get_credentials():
+        return {'password': os.getenv('TAP_POSTGRES_PASSWORD')}
+
+    def get_properties(self, original_properties=True):
+        return_value = {
+            'host' : os.getenv('TAP_POSTGRES_HOST'),
+            'dbname' : os.getenv('TAP_POSTGRES_DBNAME'),
+            'port' : os.getenv('TAP_POSTGRES_PORT'),
+            'user' : os.getenv('TAP_POSTGRES_USER'),
+            'default_replication_method' : self.FULL_TABLE,
+            'filter_dbs' : 'discovery1'
+            # 'logical_poll_total_seconds': '10',
+            # 'wal2json_message_format': '1'
```

Comment:
Should these have TODO's or should they be removed?

Reply:
Removed the poll. Set wal2json with a TODO